bundles/EcommerceFrameworkBundle/CartManager/MultiCartManager.php

```diff
@@ -130,6 +127,17 @@ protected function initSavedCarts()
         }
     }
 
+    /**
+     * @return CartInterface[]|null
+     */
+    protected function getAllCartsForCurrentUser()
+    {
+        $classname = $this->getCartClassName();
+
+        /* @var CartInterface[] $carts */
```

Comment:
There is no `$carts` variable. I think we can drop this comment

Reply:
Yes indeed!